propertyestimator/backends/dask.py

```diff
@@ -177,8 +198,14 @@ def __init__(self,
             if resources_per_worker.number_of_gpus > 1:
                 raise ValueError('Only one GPU per worker is currently supported.')
 
-        self._cluster = None
-        self._client = None
+        # For now we need to set this to some high number to ensure
```

Comment:
This is a good detail to comment on, and I'm sure this will help new people coming into the codebase.